eva-accession-core/src/test/resources/accession-test.properties

```diff
@@ -0,0 +1,3 @@
+accessioning.instanceId=test-instance-01;
```

Comment:
Is the semicolon intentional?

Reply:
hum, that came from accession-commons, I didn't realise. 

pika.. err.. @jorizci! I choose you!

Reply:
Remove it, I was probably smoking a cookie or something similar